docs/guides/getting-started-orchestrate-api-endpoints.md

```diff
@@ -19,15 +19,13 @@ To use a **REST Connector** in your process, follow the steps below:
 
 1. Create a BPMN diagram. To do this, navigate to Web Modeler via the **Modeler** tab, and click **New project**.
 2. Name your project and select **New > BPMN Diagram > + Create blank**.
-3. Give your model a descriptive name and id. On the right side of the page, expand the **General** section of the properties panel to find the name and id fields. For this guide, we'll use **API Endpoint Tutorial** for the name and **api-endpoint-tutorial** for the id.
+3. Give your model a descriptive name and id. On the right side of the page, expand the **General** section of the properties panel to find the name and id fields. For this guide, we'll use `API Endpoint Tutorial` for the name and `api-endpoint-tutorial` for the id.
 4. Use Web Modeler to design a BPMN flow with a Connector task. Create a Connector task by dragging the rectangular task element from the palette, or click the existing start event and the displayed task element to the right of the start event.
-5. Change the task type by clicking the wrench icon and select **REST Connector (No Auth)** in the **Connectors** section.
+5. Change the task type by clicking the wrench icon and select **REST Connector (No Auth)** in the **Connectors** section. Alternatively, you can directly choose a **REST Connector** by using the **append menu**.
 
-![create sendgrid connector wrench](img/connectors-rest-create-task-wrench.png)
+   ![create sendgrid connector wrench](img/connectors-rest-create-task-wrench.png)
 
-Alternatively, you can directly create a **REST Connector** by using the append menu:
-
-![create rest connector append](img/connectors-create-task-append.png)
+6. Add a descriptive name using the properties panel. For this guide, we'll use `Make a request`. Since you previously opened the **General** section of the properties panel, it is likely still open when working with your connector configuration.
```

Comment:
I'm wary of using the term "likely" -- will it still be open or will it not? If not, why not/what should you do?